src/ResourceManager/Network/Commands.Network/ExpressRouteCircuit/Peering/AzureExpressRouteCircuitPeeringConfigBase.cs

```diff
@@ -48,7 +48,7 @@ public class AzureExpressRouteCircuitPeeringConfigBase : NetworkBaseCmdlet
             Mandatory = true,
             HelpMessage = "The PeerAsn")]
         [ValidateNotNullOrEmpty]
-        public int PeerASN { get; set; }
+        public uint PeerASN { get; set; }
```

Comment:
Why not "long" as you did in the SDK?

Reply:
I think I have already explained it.  The swagger supports only int32 and int64, so sdk uses long so that it can match to the swagger.
The asn is actually a 32 bit unsigned entity, so in the powershell front end, its better to keep it as 4 byte unsigned integer.